app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> bool:
+    commands = command.split(" ")
+    if len(commands) == 3 and commands[0] == "cp":
+        cp_file = commands[1]
+        new_file = commands[2]
```

Comment:
better to use unpacking
```suggestion
        _, cp_file, new_file = commands
```